src/inapp/utils.ts

```diff
@@ -232,6 +232,48 @@ const mediaQueryLg = global?.matchMedia?.(
 );
 const mediaQueryXl = global?.matchMedia?.('(min-width: 1301px)');
 
+/**
+ *
+ * @returns { HTMLIFrameElement } iframe with sandbox and hidden styling applied for of an inapp message to render with
+ */
+const generateSecuredIFrame = () => {
+  const iframe = document.createElement('iframe');
+  iframe.setAttribute('id', 'iterable-iframe');
+  iframe.setAttribute('sandbox', 'allow-same-origin allow-popups');
```

Comment:
I think it'd be good to provide a short comment to justify why we're adding `allow-popups`, perhaps something like `allow-popups: Allows links with target="_blank" inside iframe to successfully open in a new tab.`